core-api/src/main/java/org/testng/reporters/FileStringBuffer.java

```diff
@@ -131,7 +132,7 @@ public String toString() {
     if (m_file != null) {
       flushToFile();
       try {
-        result = Files.readFile(m_file);
+        result = new String(Files.readAllBytes(m_file.toPath()));
```

Comment:
Ok, but why?

Reply:
To avoid pulling `org.testng.reporters.Files` to `core-api`
It was the only usage, so I reworked it instead of moving `Files` to `api`